build/docker/zap-api-scan.py

```diff
@@ -244,13 +247,10 @@ def main(argv):
     elif config_url:
         # load config file from url
         try:
-            load_config(urllib2.urlopen(config_url), config_dict, config_msg, out_of_scope_dict)
+            load_config(urlopen(config_url), config_dict, config_msg, out_of_scope_dict)
         except ValueError as e:
             logging.warning(e)
             sys.exit(3)
-        except:
```

Comment:
Was this removed on purpose?

Reply:
No, I got confused resolving a merge conflict.

Reply:
Fixed.